src/pages/index.js

```diff
@@ -228,76 +229,92 @@ class IndexPage extends React.Component {
             <Copy id="welcome">
               <Section>
                 <p>
-                  The Ethereum Foundation Fellowship Program is an opportunity
-                  for changemakers to use Ethereum to help solve challenges in
-                  developing countries.
-                </p>
-                <p>
-                  Changemakers drive their independent projects while working
-                  alongside like-minded Fellows. The Ethereum Foundation
-                  provides a unique experience as well as access to mentorship
-                  and technical support.
+                The Ethereum Foundation Fellowship Program is a small forum for leaders who, during the course of 6 months,
+                drive their own projects that use Ethereum as a public good destined to help billions of people coordinate and thrive.
+
                 </p>
                 <p>
-                  We believe that{" "}
-                  <Link to="https://ethereum.org/">Ethereum’s</Link> capability
-                  to build trust, transparency, and efficiency has the potential
-                  to help solve some of the challenges that exist in developing
-                  countries, and are faced by billions of people. These
-                  challenges include the lack of formal identification, lack of
-                  access to banking and financial services, and lack of internet
-                  connectivity. It is in this context - an overlap between
-                  opportunities, impact, and potential for Ethereum - that the
-                  Ethereum Foundation’s Fellowship Program was created.
+                  Ethereum has already reached <dfn>millions</dfn> of people,
+                  but to reach the next <dfn>billion</dfn>,
+                  Ethereum must overcome many gaps in representation across cultures, nationalities, and economic classes.
+                   </p>
+                   <p>
+                   The Fellowship Program is about bridging these gaps by identifying and supporting unique and talented individuals
+                   helping to enable Ethereum’s relevance and breaking down barriers to entry for those underrepresented people and communities
+                   who will become the future of web3.
                 </p>
               </Section>
+              <Link to="https://fellowship.ethereum.foundation/apply">
```

Comment:
```suggestion
              <Link to="/apply/">
```
FYI internal links shouldn't use the full URL (to avoid re-loading the page).